contracts/Blacklist.sol

```diff
@@ -6,18 +6,27 @@ import "./Ownable.sol";
 contract Blacklist is BlacklistInterface, Ownable {
 
     mapping (bytes32 => bool) blacklisted;
-
+    
+    /** @dev Add a node to the blacklist.
```

Comment:
There should be a new line where the first comment is on, for all other comments.